activities/communication/communication-remote-first.md

```diff
@@ -0,0 +1,29 @@
+---
+type: Resource
+---
+
+As we aim to work remote-first, and a lot of our work with codebase communities happens remotely, here are some tips on communication in a remote-first environment.
+
+## Questioning video-call as default. 
+
+This is definitely down to individual preferences, but video calls may be more taxing on the eyes and require more points to pay attention to. 
+
+Some people on the team found that when having 1-on-1 check-in calls or chats, doing it over a phone call while walking around outside worked just as well (or better) and was less taxing/tiring. 
+
+## 'Closing the loop' on open-ended requests.
+
+We identified a potentially harmful pattern around open requests not being 'closed'. 
+
+So for example, someone sends an open ended request for input or help or review, but doesn't hear back from anyone.
+
+This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and a cycle of the item resurfacing without resolution.
```

Comment:
```suggestion
This can be demotivating for the initiator (feeling of shouting into the void, not being taken seriously, valued or respected, and feeling like doing meaningless work), expensive for the organisation (many people reading, thinking about it, closing, opening it later again, without coming to a conclusion), and lead to a cycle of the item resurfacing without resolution.
```